test/lease_test.go

```diff
@@ -22,6 +24,53 @@ func TestLease(t *testing.T) {
 		g.Expect(resp.ID).To(Equal(clientv3.LeaseID(ttl)))
 		g.Expect(resp.TTL).To(Equal(ttl))
 	})
+
+	t.Run("UseLease", func(t *testing.T) {
+		var ttl int64 = 2
```

Comment:
any reason why we cannot have this be 1 second?
```suggestion
		var ttl int64 = 1
```